frontend/device/input.lua

```diff
@@ -147,6 +147,8 @@ function Input:init()
     -- set up fake event map
     self.event_map[10000] = "IntoSS" -- go into screen saver
     self.event_map[10001] = "OutOfSS" -- go out of screen saver
+    self.event_map[10010] = "UsbPlugIn"
```

Comment:
We didn't have this yet? I thought we already detected USB plug/unplug on Kobo but didn't take action on it.

Reply:
in Kobo we use the Charging/NotCharging events for both a computer and a wall charger.  This should be modified to enable mass storage support.

Reply:
this will be done in https://github.com/koreader/koreader-base/pull/747